src/lib/storage/session.js

```diff
@@ -235,6 +238,31 @@ SessionStorage.prototype = {
   },
 
   /**
+   * Get the selected tab index for a group.
+   *
+   * @param {ChromeWindow} chromeWindow
+   * @param {Number} groupID - the groupID
+   */
+  getGroupSelectedIndex: function(chromeWindow, groupID) {
+    let groupsData = this._getGroupsData(chromeWindow);
+    let currentGroup = groupsData[groupID];
```

Comment:
What is the preferred way of testing for a non-existent key? On the next line, I compare it to null, and return 0 in that case.